xds/src/main/java/io/grpc/xds/internal/rbac/engine/cel/Interpreter.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.xds.internal;
+
+import com.google.api.expr.v1alpha1.CheckedExpr;
+import io.grpc.xds.InterpreterException;
+
+/** Interface to a CEL interpreter. */
+public interface Interpreter {
+  /** Creates an interpretable for the given expression. */
+  Interpretable createInterpretable(CheckedExpr checkedExpr) throws InterpreterException;
```

Comment:
Since right now we can get only `Expr` instead of `CheckedExpr`, you can add another interface 
```
Interpretable createInterpretable(Expr expr) throws InterpreterException;
```
Anyway we need to do some integration work when the APIs are ready.

Reply:
Sure!